schemas/io-package.json

```diff
@@ -958,6 +958,49 @@
             ]
           }
         },
+        "javascriptRules": {
+          "type": "object",
+          "properties": {
+            "i18n": {
+              "description": "Translation for the rule block",
+              "anyOf": [
+                {
+                  "description": "Translations will be loaded from javascriptRules.url and i18n/<LANG>.json",
+                  "const": true
+                },
+                {
+                  "description": "Define translations here",
+                  "type": "object",
+                  "patternProperties": {
+                    ".+": {
+                      "$ref": "#/definitions/multilingual"
+                    }
+                  }
+                }
+              ]
+            },
+            "type": {
+              "description": "Define type='module' if rules written with TypeScript",
```

Comment:
A bit confusing because at least from a Node.JS perspective this confuses ESM and TS. However, if this is simply legacy then LGTM.